src/main/java/jssc/SerialPortEvent.java

```diff
@@ -38,15 +38,41 @@
     @Deprecated
     private String portName;
 
-    public static final int RXCHAR = 1;
-    public static final int RXFLAG = 2;
-    public static final int TXEMPTY = 4;
-    public static final int CTS = 8;
-    public static final int DSR = 16;
-    public static final int RLSD = 32;
-    public static final int BREAK = 64;
-    public static final int ERR = 128;
-    public static final int RING = 256;
+    /** Deprecated: Use <code>SerialPort.MASK_RXCHAR</code> instead **/
+    @Deprecated
```

Comment:
I've marked all of these as deprecated because they're identical to the `SerialPort` class, but since they're public, they actually be used by some projects.  I'm not sure why one class uses naming `MASK_RXCHAR` and the other uses `RXCHAR`, but I see no value in the redundancy.

This change  related to the javadocs because all `public` variables must have a comment explaining what they're for.  Thoughts welcome.